api/main_endpoints/models/password-reset.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const PasswordReset = new Schema(
+  {
+    email: {
```

Comment:
two things:
- [ ] the definition of the schema should follow the naming conventions of the other schemas
- [ ] the file name should also follow the naming convention of the other files in the models folder